docs/rule/no-yield-to-default.md

```diff
@@ -0,0 +1,77 @@
+# no-yield-to-default
```

Comment:
Still a bunch of `TODO`s in this file.

Reply:
Removed. Anyway the docs require a better description and references I think.

Reply:
@chancancode - Can you help with some prose/info here?

Reply:
Something like:

> The `yield` keyword can be used for invoking blocks passed into a component. The `to` named argument specifies which of the blocks to yield too. Specifying `{{yield to="default"}}` is unnecessary, as that is the default behavior. Likewise, `{{has-block}}` and `{{has-block-params}}` also defaults to checking the "default" block.